xray_core/src/buffer.rs

```diff
@@ -242,6 +294,17 @@ impl Buffer {
             text
         })
     }
+
+    pub fn as_register_stream(&self) -> RegisterStream<MPMCFutReceiver<Version>> {
```

Comment:
I think it would be more orthogonal to just return the raw stream in a method named `changes()`. The caller can wrap it in a `Latest` stream themselves.